assets/python/ml/pyml:data_input:read_csv:pandas.j2.py

```diff
@@ -17,18 +17,30 @@
 
 
 import pandas
-
+import sklearn.preprocessing
 import settings
 
 with settings.context as context:
     data = pandas.read_csv(settings.datafile)
 
+    # Train
+    # By default, we don't do train/test splitting; in other words, the train/test set are one and the same.
+    # Other units (such as a train/test splitter) down the line can adjust this as-needed.
     if settings.is_workflow_running_to_train:
-        # If we're training, we have an extra targets column to extract
         target = data.pop(settings.target_column_name).to_numpy()
+        if settings.is_classification:
+            target = target.astype(int)
         target = target.reshape(-1, 1)  # Reshape array to be used by sklearn
-        context.save(target, "target")
+        descriptors = data.to_numpy()
```

Comment:
I don't understand the comment on line 33 - let's elaborate maybe?

Reply:
Rephrased the comment to be more clear. This is being used to convert the vector we get from pandas into a column-vector